cmake/test.cmake

```diff
@@ -235,6 +236,10 @@ finally:
          "${CMAKE_INSTALL_PREFIX}/plugin/adsk/scripts")
     list(APPEND MAYAUSD_VARNAME_MAYA_SCRIPT_PATH
          "${CMAKE_INSTALL_PREFIX}/plugin/adsk/scripts")
+    list(APPEND MAYAUSD_VARNAME_PXR_MTLX_STDLIB_SEARCH_PATHS
+         "${PXR_USD_LOCATION}/libraries")
+    list(APPEND MAYAUSD_VARNAME_PXR_MTLX_STDLIB_SEARCH_PATHS
+         "${CMAKE_INSTALL_PREFIX}/libraries")
```

Comment:
Add these two env vars here so that every test will have them. Jonathan's new testShaderDef requires them.

Reply:
Looks good. Can you make this conditional on the presence of MaterialX (for older Maya/USD build combinations).

Reply:
I could wrap them inside of `CMAKE_WANT_MATERIALX_BUILD` - would that be good?

Reply:
Yes. that would work since test.cmake is processed after we are done setting CMAKE_WANT_MATERIALX_BUILD.

Reply:
But having extra variable does not hurt. No need for an extra check here. Sorry for the noise.